cg/server/dto/orders_request.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+
+
+class OrdersRequest(BaseModel):
+    limit: int | None = None
```

Comment:
I'd create an `OrderResponse` as well (I think we want to remove the `to_dict` logic from our database models anyway) and then a utils module in the `services/orders` dir with a method.

Reply:
OrderResponse created!